client/sites-dashboard/components/sites-site-item-thumbnail.tsx

```diff
@@ -26,7 +26,14 @@ export const SiteItemThumbnail = ( { site, ...props }: SiteItemThumbnailProps )
 		const updatedAt = new Date( site.options.updated_at );
 		updatedAt.setMinutes( 0 );
 		updatedAt.setSeconds( 0 );
-		siteUrl = addQueryArgs( siteUrl, { v: updatedAt.getTime() / 1000 } );
+		siteUrl = addQueryArgs( siteUrl, {
+			v: updatedAt.getTime() / 1000,
+
+			// This combination of flags stops free site headers and cookie banners from appearing.
+			iframe: true,
```

Comment:
Why do we need the `iframe` flag?

Reply:
Short answer is that the cookie banner didn't disappear until I added all these flags together :)

But the `iframe` flag does some other things like disabling autoplay videos 2d309-pb